modules/packages/LinearAlgebra.chpl

```diff
@@ -2418,16 +2421,16 @@ proc kron(A: [?ADom] ?eltType, B: [?BDom] eltType) {
 }
 
 /*
-  Matrix exponential using Pade approximation. This method returns N*N matrix which
-  is Matrix exponential of `A`
+  Matrix exponential using Pade approximation. This method returns square matrix which
+  is matrix exponential of ``A``.
```

Comment:
I think this is missing some articles, while you're here.  E.g. "this method returns *a* square matrix..."